### PR TITLE
[trust] Hook up August melee

### PR DESCRIPTION
**Pull request by zach2good**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

I saw a discussion get bumped about this yesterday, and if people are _still_ having trouble figuring out how to hook this up - I may as well just do it.

Damage values are crap, but he melees and animates correctly, with the correct log messages.

### CODE REVIEW

**Review by claywar — APPROVED** [review]

File: scripts/globals/mobskills/august_melee_axe.lua [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local numhits = 1
+    local accmod = 1
+    local dmgmod = 1
+    local info = xi.mobskills.mobPhysicalMove(mob, target, skill, numhits, accmod, dmgmod, xi.mobskills.magicalTpBonus.NO_EFFECT)
+    local dmg = xi.mobskills.mobFinalAdjustments(info.dmg, mob, skill, target, xi.attackType.PHYSICAL, xi.damageType.BLUNT, info.hitslanded)
+    target:takeDamage(dmg, mob, xi.attackType.PHYSICAL, xi.damageType.BLUNT)
+    skill:setMsg(xi.msg.basic.HIT_DMG)
```
Comment (TeoTwawki) [inline scripts/globals/mobskills/august_melee_axe.lua:21]: the set msg actually shouldn't be needed on melee substitute skills, the setMobSkillAttack system normally handles that automatically

Comment (zach2good) [reply scripts/globals/mobskills/august_melee_axe.lua:21]: Doesn't handle it automatically unless I have that line:
image

Comment (TeoTwawki) [reply scripts/globals/mobskills/august_melee_axe.lua:21]: Strange...Jormy Tiamat and Ouryu don't have setMsg in their scripts for their skillattacks

Comment (TeoTwawki) [reply scripts/globals/mobskills/august_melee_axe.lua:21]: now wondering if I broke that when I refactored the flags

Comment (TeoTwawki) [reply scripts/globals/mobskills/august_melee_axe.lua:21]: Appears I did. lets leave the setMsg and I'll fix the other mobs skill attack scripts later.

**Review by TeoTwawki — APPROVED** [review]

**Pull request merged**